.github/workflows/pg-migtests.yml

```diff
@@ -113,9 +125,12 @@ jobs:
 
       - name: "TEST: pg-partitions"
         run: migtests/scripts/run-test.sh pg/partitions
+        if: matrix.version != '2.16.7.0-b59'
```

Comment:
Because I have added PKs now in the partitions schema which will create some INDEX related DDLs which doesn't work in this by version, we have similar test pg-partitions-with-indexes which also doesn't run for this version